awsenergylabelercli/awsenergylabelercli.py

```diff
@@ -82,22 +89,24 @@ def get_arguments():
                         action='store',
                         dest='logger_config',
                         help='The location of the logging config json file',
-                        default='')
+                        default=os.environ.get('LOG_CONFIG', ''))
     parser.add_argument('--log-level',
                         '-L',
                         help='Provide the log level. Defaults to info.',
                         dest='log_level',
                         action='store',
-                        default='info',
+                        default=os.environ.get('LOG_LEVEL', 'info'),
                         choices=['debug',
                                  'info',
                                  'warning',
                                  'error',
                                  'critical'])
-    scope = parser.add_mutually_exclusive_group(required=True)
+    scope = parser.add_mutually_exclusive_group(required=not (os.environ.get('LANDING_ZONE_NAME') or
```

Comment:
Not sure exactly what this is supposed to be doing?

Reply:
Do you think that dynamically changing the behavior of the tool based on the environment is the most elegant way to go for this?